Ticketing/Ticketing.BAL/Enums/Statuses.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Ticketing.BAL.Enums
+{
+    public class Statuses
+    {
+        public enum PaymentStatusEnum { No_payment = 1, Part_payment = 2, Full_payment = 3, Payment_Failed = 4 };
```

Comment:
naming everywhere. You can refer MSDN: https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/identifier-names

Reply:
done